database/mongodb/README.md

```diff
@@ -13,6 +13,8 @@
 |------------|---------------------|-------------|
 | `x-migrations-collection` | `MigrationsCollection` | Name of the migrations collection |
 | `x-transaction-mode` | `TransactionMode` | If set to `true` wrap commands in [transaction](https://docs.mongodb.com/manual/core/transactions). Available only for replica set. Driver is using [strconv.ParseBool](https://golang.org/pkg/strconv/#ParseBool) for parsing|
+| `x-advisory-lock-collection` | `migrate_advisory_lock` | The name of the collection to use for advisory locking |
+| `x-advisory-lock-backoff-seconds` | `15` | How max time that the advisory lock will wait during exponential backoff if the db is already locked. |
```

Comment:
I think the default might want to be something like 300, we should submit the PR and ask upstream though about it. Don't change it now.

Reply:
I think the default might want to be something like 300, we should submit the PR and ask upstream though about it. Don't change it now.